scripts/variantstore/wdl/GvsImportGenomes.wdl

```diff
@@ -217,6 +219,7 @@ task LoadData {
     Boolean? drop_state_includes_greater_than = false
     Boolean force_loading_from_non_allele_specific = false
     Boolean skip_loading_vqsr_fields = false
+    Boolean use_compressed_references = true
```

Comment:
Why is it defaulted to true here?

Reply:
Initial, small-scale testing.  I thought I swept over the code and changed all arguments to be false by default, but I must've missed this one.  Nice catch!  I'll change it now